Source/WebCore/PAL/pal/text/TextEncodingRegistry.h

```diff
@@ -43,6 +43,7 @@ PAL_EXPORT std::unique_ptr<TextCodec> newTextCodec(const TextEncoding&);
 
 // Only TextEncoding should use the following functions directly.
 ASCIILiteral atomCanonicalTextEncodingName(const char* alias);
```

Comment:
Do we still need this one?

Reply:
I think so because I was hoping to remove it and couldn't easily. But I'll double check.

Reply:
Yes, if I drop the constructor above, I can drop this one too.